src/Widgets/UserBox.vala

```diff
@@ -22,54 +22,60 @@ public class Session.Widgets.Userbox : Gtk.ListBoxRow {
     private const string LOGGED_OFF = _("Logged out");
     private const int ICON_SIZE = 48;
 
-    public Act.User? user { public get; private set; }
-    public bool is_guest = false;
+    public Act.User? user { get; set; }
```

Comment:
The user should not be modifiable from the outside of the class, so it should be `{ get; construct; }`